src/sharepoint/order/sharepoint-order.html

```diff
@@ -1,125 +1,90 @@
 
-                        <div class="clearfix"
-                             data-ng-if="orderCtrl.accountsIds">
-                            <div data-pagination-front
-                                 data-items="orderCtrl.accountsIds"
-                                 data-paginated-items="orderCtrl.accountsList"
-                                 data-current-page="currentPage"
-                                 data-items-per-page="itemsPerPage"
-                                 data-nb-pages="nbPages"
-                                 data-transform-item="orderCtrl.onTranformItem(item)"
-                                 data-on-transform-item-done="orderCtrl.onTranformItemDone()"
-                                 data-page-placeholder="{{ 'pagination_page' | translate: { current: currentPage, last: nbPages } }}"
-                                 data-item-per-page-placeholder="{{ 'pagination_display' | translate }}">
-                            </div>
-                        </div>
+                        <oui-datagrid id="exchangeAccountsDatagrid"
+                                      data-rows-loader="$ctrl.getAccounts()"
+                                      data-row-loader="$ctrl.getAccount($row)"
+                                      data-on-row-select="$ctrl.onAccountsSelected($rows)"
+                                      data-selectable-rows
+                                      data-empty-placeholder="{{:: 'sharepoint_accounts_empty' | translate }}">
+                            <oui-column data-title=":: 'sharepoint_accounts_userPrincipalName' | translate"
+                                        data-property="primaryEmailAddress">
+                            </oui-column>
+                        </oui-datagrid>
                     </div>
 
                     <div class="my-5">
-                        <a class="oui-button oui-button_primary oui-button_large-height" target="_blank" title="{{ 'sharepoint_order_button_label' | translate }} ({{ 'core_new_window' | translate }})"
-                           data-translate="sharepoint_order_button_label"
-                           data-ng-href="{{orderCtrl.getSharepointOrderUrl()}}"
-                           data-ng-if="!orderCtrl.associateExchange"
-                           data-ng-show="orderCtrl.activateSharepointForm.$valid && orderCtrl.standAloneQuantity">
-                        </a>
-                        <a class="oui-button oui-button_primary oui-button_large-height" target="_blank" title="{{ 'sharepoint_order_button_label' | translate }} ({{ 'core_new_window' | translate }})"
-                           data-translate="sharepoint_order_button_label"
-                           data-ng-href="{{orderCtrl.getSharepointOrderUrl()}}"
-                           data-ng-if="orderCtrl.associateExchange"
-                           data-ng-show="orderCtrl.accountsToActivate.length > 0">
-                        </a>
+                        <oui-button
```

Comment:
Don't use a `button` for hyperlinks, use a `a`

Reply:
This button can be disabled and we don't know the URL at first.